packages/vest/src/core/suite/produce/summarySelectors/suiteSelectors.ts

```diff
@@ -80,12 +84,32 @@ export function suiteSelectors(summary: SuiteSummary): SuiteSelectors {
 
   // Responses
 
+  function getWarning(): FailureMessage;
+  function getWarning(fieldName: string): string;
+  function getWarning(fieldName?: string): GetFailureResponse {
```

Comment:
I don't think there's need for an overload here (and same with getError).
fieldName should be mandatory, as without a fieldName, we don't really know which field should yield the "one" message.
```suggestion
  function getWarning(fieldName: string): string;
```